specification/cosmos-db/resource-manager/readme.go.md

```diff
@@ -21,6 +21,7 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2021-07-preview
```

Comment:
Please assign a `output-folder` for this tag. You can take the entries below as reference

Reply:
Done